src/main/java/git/GitCloner.java

```diff
@@ -22,9 +22,9 @@ public static void downloadRepo(String organization, String repoName, String bra
         }
 
         try {
-            GitChecker.checkout(FileUtil.getRepoDirectory(organization, repoName), branchName);
+            GitChecker.checkout(FileUtil.getRepoDirectory(organization,repoName), branchName);
```

Comment:
uh... last one, there should be a space after comma. At the same time, you can also use checkstyle plugin in the intellij by following [this guide](https://github.com/se-edu/addressbook-level4/blob/master/docs/UsingCheckstyle.adoc)